<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="0"
            failures="1"
            tests="1">

  <testsuite name="grid"
    errors="0" failures="1" hostname="" id="" package="grid" skipped="0"
    tests="1" time="12.65" timestamp="Wed, 22 Jul 2015 10:49:41 GMT">
  
    <testcase name="Grid Test" time="12.65" assertions="3">

      <failure message="Testing if element &lt;.mdl-grid&gt; is present.">Testing if element &lt;.mdl-grid&gt; is present.: Expected &quot;present&quot; but got: &quot;null&quot;
    at Object.&lt;anonymous&gt; (C:\Users\user\AppData\Roaming\npm\node_modules\nightwatch\lib\core\assertion.js:105:19)
    at HttpRequest.&lt;anonymous&gt; (C:\Users\user\AppData\Roaming\npm\node_modules\nightwatch\lib\index.js:311:20)
    at HttpRequest.emit (events.js:110:17)
    at HttpRequest.&lt;anonymous&gt; (C:\Users\user\AppData\Roaming\npm\node_modules\nightwatch\lib\index.js:344:15)
    at HttpRequest.emit (events.js:118:17)
    at IncomingMessage.&lt;anonymous&gt; (C:\Users\user\AppData\Roaming\npm\node_modules\nightwatch\lib\http\request.js:150:16)
    at IncomingMessage.emit (events.js:129:20)
    at _stream_readable.js:908:16
    at process._tickCallback (node.js:355:11)</failure>

      <failure message="Testing if element &lt;.mdl-cell&gt; is present.">Testing if element &lt;.mdl-cell&gt; is present.: Expected &quot;present&quot; but got: &quot;null&quot;
    at Object.&lt;anonymous&gt; (C:\Users\user\AppData\Roaming\npm\node_modules\nightwatch\lib\core\assertion.js:105:19)
    at HttpRequest.&lt;anonymous&gt; (C:\Users\user\AppData\Roaming\npm\node_modules\nightwatch\lib\index.js:311:20)
    at HttpRequest.emit (events.js:110:17)
    at HttpRequest.&lt;anonymous&gt; (C:\Users\user\AppData\Roaming\npm\node_modules\nightwatch\lib\index.js:344:15)
    at HttpRequest.emit (events.js:118:17)
    at IncomingMessage.&lt;anonymous&gt; (C:\Users\user\AppData\Roaming\npm\node_modules\nightwatch\lib\http\request.js:150:16)
    at IncomingMessage.emit (events.js:129:20)
    at _stream_readable.js:908:16
    at process._tickCallback (node.js:355:11)</failure>

    
    </testcase>
  
  
  </testsuite>
</testsuites>
